src/utils/request/createResponseFromIsomorphicResponse.ts

```diff
@@ -1,34 +1,11 @@
-import { encodeBuffer, IsomorphicResponse } from '@mswjs/interceptors'
-
-const noop = () => {
-  throw new Error('Not implemented')
-}
+import { IsomorphicResponse } from '@mswjs/interceptors'
 
 export function createResponseFromIsomorphicResponse(
   response: IsomorphicResponse,
 ): Response {
-  return {
-    ...response,
-    ok: response.status >= 200 && response.status < 300,
-    url: '',
-    type: 'default',
+  return new Response(response.body, {
```

Comment:
It scares me a bit why the previous logic was contrived here. This function is only called in the fallback mode, which is browser-only if the Service Worker API is not available (old browsers, locally-served apps). Since we are in the browser, we can rely on `Response`, we always could.